.gitignore

```diff
@@ -64,7 +64,6 @@ src/ext/http-parser/
 src/ext/jansson/
 src/ext/json/
 src/ext/libevent/
-src/ext/libmaxminddb/
```

Comment:
Since some more changes are to be performed in any case, would you mind making sure that we don't list in `.gitignore` stuff that - after recent changes - we don't put in `src/ext` anymore?